### ISSUE TITLE
Support .swig extension files

### ISSUE THREAD

**Issue by Toub** [abdelouahabb/swig-brackets#1]
Only .html is currently supported.

Could you add support for .swig files?

**Comment by abdelouahabb** [comment]
hi, I did not know about that, do you mean:

`fileExtensions: ["html", "swig"]` ?

**Comment by Toub** [comment]
Exactly.
Because https://github.com/petecoop/generator-express uses .swig extension by default.

**Comment by Toub** [comment]
@see https://github.com/petecoop/generator-express/tree/master/app/templates/views/swig

**Comment by abdelouahabb** [comment]
can you try it now?

https://github.com/abdelouahabb/swig-brackets/commit/877b03e407e4f2bf0b2b35eb9b3ec38681987dc5

and if you have some new extension, feel free to add pulls ^_^ am not a real node.js programmer, I use python, so I just made this extension because I found that it has the same extension as Django to help people that uses it.

**Comment by Toub** [comment]
Works!

But you should update the revision number (currently "version": "0.0.1").

**Comment by abdelouahabb** [comment]
no, it is on the dev branch, so now i will make the 0.0.2 since it works there ^_^
